CONTRIBUTING.MD

```diff
@@ -0,0 +1,81 @@
+# Contributing guide
+
+Thanks for your contribution 🥳 to the Fastily website redesign project!
+
+[Code of Conduct](https://github.com/fastify/.github/blob/main/CODE_OF_CONDUCT.md)
+
+* [Vocabulary](#vocabulary)
+* [📝 Commit message guidelines](#-commit-message-guidelines)
+* [📜 Pull Request Policy](#-pull-request-policy)
+* [Before merging](#before-merging)
+* [When merging](#when-merging)
+* [Becoming a collaborator](#becoming-a-collaborator)
+* [🐛 Debugging failing checks](#-debugging-failing-checks)
+* [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin-11)
+
+## 📝 Commit message guidelines
+
+This project follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
+
+Basic rules:
+
+Supported subsystems:
+> build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test
+
+* Commit messages **must** start with a capital letter
```

Comment:
Please, relax this section

In the fastify org, we don't apply such strictness

As written in issue #32 
I would just link the main repo's contribution page at https://github.com/fastify/fastify/blob/main/CONTRIBUTING.md
and add more information for new joiners

Reply:
I will embellish this